cogniac/edgeflow.py

```diff
@@ -222,16 +222,28 @@ def upload():
     #
     # -------------------------------------------------------------------------
 
-    def flush_upload_queue(self, start_time=None, stop_time=None):
+    def time_bound_media_upload(self, start_time, end_time):
```

Comment:
It might be convenient to have an end_time=None default under which condition we set as current time() for user.